Make Definition Clearer (#35178)
freeCodeCamp/freeCodeCamp: guide/english/html/attributes/required/index.md

## Required

> The HTML `required` attribute specifies that an input field must be filled out before a form can be submitted. This means that the user must fill in the field. <br /> 

Reviewer: ```suggestion
The HTML `required` attribute specifies that an input field must be compulsorily filled out before a form can be submitted. <br /> 
```

Reviewer 2: @thecodingaviator While this does shorten the text, `compulsorily` is not a word that everyone uses in their everyday life.  May be something like:
```suggestion
The HTML `required` attribute specifies that filling out an input field is mandatory before a form can be submitted. <br />
```

Reviewer: @RandellDawson it definitely does. Compulsprily is not something many would be familiar with
